.github/workflows/publish-release.yml

```diff
@@ -85,41 +86,41 @@ jobs:
         replace: " "
         include: "**pubspec.yaml"
 
-    - name: Package common
+    - name: Package realm_common
       run: |
-        cp -Lr ../common .
-        cd common
+        cp -Lr ../package/realm_common .
+        cd realm_common
         dart pub publish --dry-run || true
       working-directory: release
 
-    - name: Package generator
+    - name: Package realm_generator
       run: |
-        cp -Lr ../generator .
-        cd generator
+        cp -Lr ../packages/realm_generator .
+        cd realm_generator
         dart pub publish --dry-run || true
       working-directory: release
 
     # realm_flutter has symlinks to native binaries which should not be packaged
     - name: Cleanup binary symlinks
       run: |
-        rm -rf flutter/realm_flutter/android/src/main/cpp/lib
-        rm flutter/realm_flutter/ios/src
-        rm flutter/realm_flutter/ios/realm_dart.xcframework
-        rm flutter/realm_flutter/linux/binary
-        rm flutter/realm_flutter/windows/binary
-        rm example/binary
+        rm -rf packages/realm/android/src/main/cpp/lib
+        rm packages/realm/ios/src
+        rm packages/realm/ios/realm_dart.xcframework
+        rm packages/realm/linux/binary
+        rm packages/realm/windows/binary
```

Comment:
I know you just updated some paths, but I can see we're not removing the macos binary symlink and it's not obvious to me why.

Reply:
🤔 .. seems like an inherited mistake. 

It doesn't make it into the pub packages (or the packages.tar.gz) though. Just checked 1.9.0.
I suppose we are "saved" by the way tar handles soft links pointing outside the folder, so all these removes are really redundant? 

Anyway, goes back to my question, why are we even tarball'ing the packages and storing on s3?

Reply:
No that doesn't compute, since we use `cp -Lr` which follows links. It is not removed, because there is no symlink to remove for macos.

Still, why upload these to both pub.dev and s3? I know, I should know, but it currently escapes me 😄

Reply:
We upload to S3 for prerelease builds. Since the release workflow is running the Staging part first, regardless of whether we deploy to production or not, those will be uploaded to S3. I believe these symlinks point into the void, which is why we wanted to remove them, though it was a long time ago and it's a bit fuzzy.

Reply:
You can push pre-release packages to pub.dev as well. This makes it much easier for end-users to consume them. Also, it we don't like that, we can run a private pub.dev server.. as I recall it is even using mongodb 😄 

Anyway, I haven't changed the fact that we upload to S3.. just something to consider in the future.